jest.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   testEnvironment: 'jsdom',
-  testMatch: ['**/src/**/*.test.tsx'],
+  testMatch: ['**/src/**/*.test.(ts|tsx)'],
```

Comment:
nit: You'll find only `tsx` files used in the entire repo atm outside of typedefs. This was deliberate to just have only one way to make a file :-).

Reply:
Ah I didn't realise it was deliberate. I personally prefer the distinction, which is why I added them, as then you will know by looking at the file name if it contains any JSX in it or it's pure TS.